policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/subtractions/retirement_income_exemption/vt_retirement_income_exemption.py

```diff
@@ -0,0 +1,85 @@
+        # Filer can choose from one of Social Security,
+        # Civil Service Retirement System (CSRS), Military Retirement Income
+        # or other eligible retirement systems
+        # to determine eligibility and calculate retirement income exemption.
+
+        # Get social security amount
+        tax_unit_taxable_social_security = tax_unit(
+            "tax_unit_taxable_social_security", period
+        )
+        # Get vt retirement income exclusion from military retirement system
+        vt_military_retirement_pay_exclusion = tax_unit(
+            "vt_military_retirement_pay_exclusion", period
+        )
+        # Get vt retirement income exclusion from CSRS
+        vt_csrs_retirement_pay_exclusion = tax_unit(
+            "vt_csrs_retirement_pay_exclusion", period
+        )
+        # Assume that filers will always choose the largest reitrement income
+        # exclusion from various retirement system
+        larger_retirement_income = max_(
+            tax_unit_taxable_social_security,
+            vt_military_retirement_pay_exclusion,
+        )
+        chosen_retirement_income = max_(
+            larger_retirement_income, vt_csrs_retirement_pay_exclusion
+        )
+        filing_status = tax_unit("filing_status", period)
+        agi = tax_unit("adjusted_gross_income", period)
+        # Get which retirement system the filer use
+        use_ss = tax_unit_taxable_social_security == chosen_retirement_income
+        use_csrs = vt_csrs_retirement_pay_exclusion == chosen_retirement_income
+        use_military_retirement = (
+            vt_military_retirement_pay_exclusion == chosen_retirement_income
+        )
+        # Get which parameter file to use
+        root_p = parameters(
+            period
+        ).gov.states.vt.tax.income.agi.retirement_income_exemption
+        p = where(
+            use_ss, root_p.social_security.reduction, root_p.csrs.reduction
+        )
+        # List of fully qualified tax unit (SECTION I Q3)
+        fully_qualified = (agi < p.start[filing_status]) & (
+            chosen_retirement_income != 0
+        )
+        # List of partial qualified tax unit(SECTION II)
+        partial_qualified = (
+            (agi >= p.start[filing_status])
+            & (agi < p.end[filing_status])
+            & (chosen_retirement_income != 0)
+        )
+        # Calculate the exemption ratio
+        partial_exemption_ratio = max_(p.end[filing_status] - agi, 0) / (
+            root_p.divisor
+        )
+        # Round the exemption ratio to two decimal point
+        partial_exemption_ratio = round_(partial_exemption_ratio, 2)
+        # The exemption ratio should be below one
+        partial_exemption_ratio = min_(partial_exemption_ratio, 1)
+        # Calculate parital exemption amount
+        partial_exemption = chosen_retirement_income * partial_exemption_ratio
+        # Return final exemption amount based on eligibility status
+        return select(
```

Comment:
what's the default value? is anyone neither partial nor fully qualified? if not change to a where

Reply:
Hi Max. There is no default value and I have changed it to where statement.